mongo/integration/unified/entity.go

```diff
@@ -95,7 +120,98 @@ func (em *entityMap) addChangeStreamEntity(id string, stream *mongo.ChangeStream
 	return nil
 }
 
-func (em *entityMap) addEntity(ctx context.Context, entityType string, entityOptions *entityOptions) error {
+func (em *EntityMap) addErrorsEntityIfDoesntExist(id string) error {
+	// Error if a non-error entity exists with the same name
```

Comment:
But an error entity with the same name is fine? How come we don't use `verifyEntityDoesNotExist()` here?